qcodes/logger/logger.py

```diff
@@ -212,30 +210,26 @@ def start_logger() -> None:
     logging.captureWarnings(capture=True)
 
     if qc.config.telemetry.enabled:
-
-        from applicationinsights import channel
-        from applicationinsights.logging import enable
-
-        # the telemetry_handler can be flushed
-        global telemetry_handler
+        # Transport module of opencensus-ext-azure logs info 'transmission
+        # succeeded' which is also exported to azure if AzureLogHandler is
+        # in root_logger. The following lines stops that.
+        logging.getLogger('opencensus.ext.azure.common.transport').setLevel(
+            logging.WARNING)
```

Comment:
Cool thinking. But we are just trying to suppress one line of log.info(' transmission succeed'), I do not see them changing them to log.warning or anything else because it is just an info that transmission succeeded. 

Another issue is that sometime people will type start all logging more than once without closing previous logging and each time this will bump it up.

Reply:
oh, i see. then this a case of overthinking from my side :)